### DISCUSSION TITLE
Ability to swap out persistent volume

### DISCUSSION THREAD

**Discussion by virasana** [Opster/opensearch-k8s-operator#877] [General]
I would like to swap out the persistent volumes for the purposes of disaster recovery and/or blue-green releases.  
How would I re-jig the persistent volumes to point to another disk.  I am using Azure cloud. 
As the operator is the source of truth, I would be loathe to change the manifests.

Please could you advise how best to swap out the data for another disk (I don't want to delete or rename existing disks).

Many Thanks

**Comment by swoehrl-mw** [comment]
Hi @virasana. I don't think this is possible with the operator. 
In terms of disaster recovery the go-to way is doing snapshots onto external storage (for example azure storage accounts).
